pkg/cmd/kube-router.go

```diff
@@ -180,7 +180,7 @@ func (kr *KubeRouter) Run() error {
 	}
 
 	// Handle SIGINT and SIGTERM
-	ch := make(chan os.Signal)
+	ch := make(chan os.Signal, 1)
```

Comment:
This adds buffering to the channel, which was recommended. See: https://golang.org/pkg/os/signal/#Notify